tests/test/baseledger.test.js

```diff
@@ -153,11 +153,11 @@ describe('attestations observed', async function() {
 
     parsedResponse = JSON.parse(accountBalance.text);
 
-    // check that balance increased by 1
+    // check that balance is increased
     const workTokenBalanceAfter = parsedResponse.balance.amount;
     console.log('work tokens after deposit ', workTokenBalanceAfter);
 
-    expect(+workTokenBalanceAfter).to.be.equal(+workTokenBalanceBefore + 1);
+    expect(+workTokenBalanceBefore).to.be.below(+workTokenBalanceAfter);
```

Comment:
do we want to assert exact increase amount here?

Reply:
not possible due to price change